### DISCUSSION TITLE
Does anyone have any idea for a new bookmarklet hack?

### DISCUSSION THREAD

**Discussion by tran733** [3kh0/ext-remover#247] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm kinda confused on if anyone is considering to create a new bookmarklet hack. If echo is not going to create one. Does anyone have any idea for a new bookmarlet hack?

**Comment by Theblackmamba824** [comment, marked as answer]
its being made and they said it will prob b ready in a month but dont put ur hopes up
